python/paddle/fluid/executor.py

```diff
@@ -179,6 +179,16 @@ def to_name_str(var):
     return str(feed_var_names + fetch_var_names)
 
 
+class PreparedContext(object):
+    def __init__(self, handle, program, fetch_list, feed_var_name,
+                 fetch_var_name):
+        self.handle = handle
+        self.program = program
+        self.fetch_list = fetch_list
+        self.feed_var_name = feed_var_name
+        self.fetch_var_name = fetch_var_name
```

Comment:
Are all of them public members?

Reply:
yes

Reply:
Remove this class if it's no longer used?